python/tests/unit/contents/test_chat_history.py

```diff
@@ -184,14 +185,7 @@ def test_serialize():  # ignore: E501
     assert json_str is not None
     assert (
         json_str
-        == '{\n    "messages": [\n        {\n            \
-"inner_content": null,\n            "ai_model_id": null,\n            \
-"metadata": {},\n            "role": "system",\n            \
-"content": "a test system prompt",\n            "encoding": null\n        },\
-\n        {\n            "inner_content": null,\n            \
-"ai_model_id": null,\n            "metadata": {},\n            \
-"role": "user",\n            "content": "Message",\n            \
-"encoding": null\n        }\n    ]\n}'
+        == '{\n    "messages": [\n        {\n            "metadata": {},\n            "type": "ChatMessageContent",\n            "role": "system",\n            "content": "a test system prompt"\n        },\n        {\n            "metadata": {},\n            "type": "ChatMessageContent",\n            "role": "user",\n            "content": "Message"\n        }\n    ],\n    "message_type": "ChatMessageContent"\n}'  # noqa: E501
```

Comment:
Do you think it's necessary to include the metadata in the dump if there's nothing there?

Reply:
this behaviour doesn't change, just formatted differently, a {} is not None, so it shows up with model_dump...